src/tests/suite/dfuse_test.c

```diff
@@ -654,9 +787,11 @@ main(int argc, char **argv)
 					       {"metadata", no_argument, NULL, 'm'},
 					       {"directory", no_argument, NULL, 'd'},
 					       {"lowfd", no_argument, NULL, 'l'},
+					       {"exec", no_argument, NULL, 'e'},
+					       {"verifyenv", no_argument, NULL, 'v'},
 					       {NULL, 0, NULL, 0}};
 
-	while ((opt = getopt_long(argc, argv, "aM:imsdl", long_options, &index)) != -1) {
+	while ((opt = getopt_long(argc, argv, "aM:imsdlev", long_options, &index)) != -1) {
```

Comment:
You can support the long option only by leaving the short option out of this string.
```suggestion
	while ((opt = getopt_long(argc, argv, "aM:imsdle", long_options, &index)) != -1) {
```

Reply:
Thank you! I will fix it as you suggested.